src/containers/Application/ApplicationCreateNEW.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Divider, Header, Message, Segment } from 'semantic-ui-react'
+import evaluate from '@openmsupply/expression-evaluator'
+import Markdown from '../../utils/helpers/semanticReactMarkdown'
+import { ApplicationHeader, ApplicationSelectType, Loading } from '../../components'
+import { useApplicationState } from '../../contexts/ApplicationState'
+import { useUserState } from '../../contexts/UserState'
+import useCreateApplication from '../../utils/hooks/useCreateApplication'
+import useLoadTemplate from '../../utils/hooks/useLoadTemplateNEW'
+import { useRouter } from '../../utils/hooks/useRouter'
+import strings from '../../utils/constants'
+import { EvaluatorParameters } from '../../utils/types'
+import SectionsList from '../../components/Sections/SectionsList'
+
+const ApplicationCreateNEW: React.FC = () => {
+  const {
+    applicationState: { serialNumber },
+    setApplicationState,
+  } = useApplicationState()
+  const {
+    push,
+    query: { type },
+  } = useRouter()
+  const [startMessageEvaluated, setStartMessageEvaluated] = useState('')
+
+  const { error, loading, template, sections, elementsIds } = useLoadTemplate({
+    templateCode: type,
+  })
+
+  const {
+    userState: { currentUser },
+  } = useUserState()
+
+  // If template has no start message, go straight to first page of new application
+  useEffect(() => {
+    if (template && !template.startMessage) handleCreate()
+    else evaluateMessage()
+  }, [template])
+
+  const { processing, error: creationError, create } = useCreateApplication({
+    onCompleted: () => {
+      if (serialNumber && sections && sections.length > 0) {
+        // Call Application page on first section
+        const firstSection = sections[0].code
+        // The pageNumber starts in 1 when is a new application
+        push(`${serialNumber}/${firstSection}/Page1`)
+      }
+    },
+  })
+
+  const evaluateMessage = async () => {
+    const evaluatorParams: EvaluatorParameters = {
+      objects: { currentUser },
+      APIfetch: fetch,
+    }
+    evaluate(template?.startMessage || '', evaluatorParams).then((result: any) =>
+      setStartMessageEvaluated(result)
+    )
+  }
```

Comment:
I made a similar comment on your SubmissionPage PR: do you think we should do the evaluation in the 'useLoadTemplate' hook? We're doing most other evaluations inside hooks, so kinda makes sense to return "start message" pre-evaluated?

Reply:
Yes, I agree. Can I do this in a separated PR after this is merged? I think it would be tidier.